app/app.js

```diff
@@ -12,21 +13,10 @@ function Chicken (chix)  {
 }
 
 Chicken.prototype.toHtml = function() {
-  var $newChicken =  $('article.template').clone().removeClass('template');
-  var daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
-  $newChicken.attr('data-name', this.title);
-  $newChicken.attr('data-startlaying', this.startLaying);
-  $newChicken.attr('data-temper', this.temperament);
-  $newChicken.find('h3').text(this.title);
-  $newChicken.find('#1').text("Egg Production: " + this.eggProduction);
-  $newChicken.find('#2').text("Temperament: " + this.temperament);
-  $newChicken.find('#3').text("Starts Laying Eggs By: " + this.startLaying);
-  $newChicken.find('#4').text("Average Weight: " + this.weight);
-  $newChicken.find('.publishedOn').text("Published on " + this.publishedOn +" (" + daysAgo +" days ago)");
-  $newChicken.find('.description').html(this.description);
-  $newChicken.find('.head-shot img').attr('src',  this.image);
-
-  return $newChicken;
+  this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+  var template = $('#chickenTemplate').html();
+  var compiledTemplate = Handlebars.compile(template);
+  $('#chickens').append(compiledTemplate(this));
```

Comment:
This function is doing an append of the complied template data. There is also an append being done in the forEach() function on line 43. Ideally, you should only do it once (one is actually ignored), and the toHtml() function you should "return" the compiled Template data.